projects/plugins/social/src/js/components/social-module-toggle/index.tsx

```diff
@@ -59,6 +60,16 @@ const SocialModuleToggle: React.FC = () => {
 					{ __( 'Manage social media connections', 'jetpack-social' ) }
 				</Button>
 			) }
+			{ services.map( service =>
```

Comment:
We don't want to do this here, but in the modal we'll popup displaying the service icons.